wafflehaus/nova/networking/network_count_check.py

```diff
@@ -119,7 +119,7 @@ def _get_networks(body):
         networks = body.get("networks")
         if networks is None:
             return None
-        return [n["uuid"] for n in networks]
+        return [n["uuid"] for n in networks if "uuid" in n]
```

Comment:
Doesn't this allow attaching to more than the configured number of networks if its passing in ports?